lib/features/email/domain/usecases/get_email_content_interactor.dart

```diff
@@ -12,31 +13,41 @@ class GetEmailContentInteractor {
 
   GetEmailContentInteractor(this.emailRepository);
 
-  Stream<Either<Failure, Success>> execute(AccountId accountId, EmailId emailId, String? baseDownloadUrl) async* {
+  Stream<Either<Failure, Success>> execute(AccountId accountId, EmailId emailId, String? baseDownloadUrl, {EmailLoaded? emailLoaded}) async* {
     try {
       yield Right<Failure, Success>(GetEmailContentLoading());
-      final email = await emailRepository.getEmailContent(accountId, emailId);
 
-      if (email.emailContentList.isNotEmpty) {
-        final newEmailContents = await emailRepository.transformEmailContent(
-            email.emailContentList,
-            email.allAttachments.listAttachmentsDisplayedInContent,
-            baseDownloadUrl,
-            accountId);
-        final newEmailContentsDisplayed = kIsWeb
-            ? await emailRepository.addTooltipWhenHoverOnLink(newEmailContents)
-            : newEmailContents;
+      if(emailLoaded != null) {
+        await Future.delayed(const Duration(milliseconds: 300));
```

Comment:
why need delayed at here? IMO, It is not recommended to do so in interactor

Reply:
ah, shoudl `consumeState` directly in `controller`

Reply:
using `getState`